server/service.go

```diff
@@ -512,6 +534,8 @@ func (m *BoostService) handleGetPayload(w http.ResponseWriter, req *http.Request
 	// Wait for all requests to complete...
 	wg.Wait()
 
+	// send bid and signed block to relay monitor
+	go m.sendAuctionTranscriptToRelayMonitors(AuctionTranscript{Bid: originalBid.response.Data, Acceptance: payload})
```

Comment:
Since `sendAuctionTranscriptToRelayMonitors` is just spawning goroutines, I'd consider just calling it synchronously

Reply:
the calls to the monitor are spawned in goroutines, so slow monitors wouldn't affect this call (mateusz mentioned only doing this call sync, but still spawning the individual calls as goroutines)